ProjectConfig/channels.json

```
@@ -378,6 +378,32 @@
             }
         }
     },
+    "898335291008487464": {
+        "name": "Server: Tyler Hobbs (879488624197001287), CH: fidenza-owners",
+        "projectBotHandlers": {
+            "default": "fidenzaBot",
+            "stringTriggers": {
+                "icBot": [
+                    "ic",
+                    "incomplete",
+                    "control"
+                ]
+            }
+        }
+    },
+    "929413828968611880": {
+        "name": "Server: Tyler Hobbs (879488624197001287), CH: incomplete-control",
+        "projectBotHandlers": {
+            "default": "fidenzaBot",
+            "stringTriggers": {
+                "icBot": [
+                    "ic",
+                    "incomplete",
+                    "control"
+                ]
```

do you want to have icBot be the default, and fidenzaBot be the stringTriggers bot (since this channel is for incomplete controls)?
```suggestion
            "default": "icBot",
            "stringTriggers": {
                "fidenzaBot": [
                    "fidenza"
                ]
```